### ISSUE TITLE
Error Message After Install

### ISSUE THREAD

**Issue by KitchM** [syvaidya/openstego#9]
Am using Xubuntu 14.04.5 and installed openstego.  Received following message when attempting to start if from the menu item (found under Graphics):

> Failed to execute command "/usr/share/openstego/openstego".
Failed to execute child process "/usr/share/openstego/openstego" (No such file or directory)

What should be done to correct this problem?

Thank you.

**Comment by syvaidya** [comment]
Did you install it using ".deb" file?

**Comment by KitchM** [comment]
Yes, and it brought up Ubuntu Software Center to handle the install.

**Comment by danarmstrong** [comment]
The file /usr/share/openstego/openstego has a couple of issues. First is that it is not executable by default and the second is that it has Windows line-endings. To get the application to work:

```
sudo dos2unix /usr/share/openstego/openstego
sudo chmod +x /usr/share/openstego/openstego
```

If dos2unix isn't installed just use apt to install it

```
sudo apt install dos2unix
```

One those two quick steps are finished you can launch openstego from the menu entry.

**Comment by KitchM** [comment]
Okay, dos2unix was available from the Ubuntu Software Center and it was a simple install.  The two command-line commands appeared to work correctly.

However, there is a window sizing issue.  The program opens with the window short and very wide; it is wider than the screen on both sides.  Worse, none of the necessary sizing handles are functional, nor does the top right of the title bar have the proper controls to maximize or normalize the window.

**Comment by danarmstrong** [comment]
I had not noticed that until you mentioned it. I'm running at 1920x1080 resolution so the window is entirely visible for me (albeit seemingly wider than need be), but you are correct about the resizing handles not being functional.

**Comment by KitchM** [comment]
Yeah, so many programmers fail to code according to usability standards.  It drives me crazy.  But it is doubly bad when they don't even use their code correctly and don't test their widgets to see if they function.

**Comment by syvaidya** [comment]
@KitchM Majority of downloads are for windows platform and it seems to work fine. I don't have any linux setup to test this out. It would be more constructive, if you can provide a patch to fix this. Otherwise you'll just have to wait till I get time - I have full time job and a family to take care of.

**Comment by KitchM** [comment]
Thanks much for the response.

No rush at all.  Appreciate your situation.

Thanks again.

**Comment by syvaidya** [comment]
Both file permissions and window resize issues are fixed now with v0.7.1.
